<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SumedhKrishna" failures="4" tests="4" name="FIREFOX(failed)" time="49.039" errors="0" timestamp="2023-09-05T22:26:38 EDT">
  <testcase classname="com.tekarck.testCases.TC04B_InvalidUserNameAndPassword" name="loginToAppWithInvalidCreds" time="19.951">
    <failure type="java.lang.AssertionError" message="expected [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Please check your username and password. If you still can't log in, contact your Salesforce administrator.] but found [null]
at com.tekarck.testCases.TC04B_InvalidUserNameAndPassword.loginToAppWithInvalidCreds(TC04B_InvalidUserNameAndPassword.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginToAppWithInvalidCreds -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.536">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@tek.com  Sumedh expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@tek.com  Sumedh expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="5.442">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@tek  Sumedh@03 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@tek  Sumedh@03 expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
  <testcase classname="com.tekarck.testCases.TC02_ValidLoginToSalesForce" name="loginToSalesForceAppUsingDataProviders" time="10.5">
    <failure type="java.lang.AssertionError" message="Failed to login for User Name and password : : anusha@t.com  Anusha expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed to login for User Name and password : : anusha@t.com  Anusha expected [true] but found [false]
at com.tekarck.testCases.TC02_ValidLoginToSalesForce.loginToSalesForceAppUsingDataProviders(TC02_ValidLoginToSalesForce.java:178)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginToSalesForceAppUsingDataProviders -->
</testsuite> <!-- FIREFOX(failed) -->
